### ISSUE TITLE
cannot found  lightdm-webkit-greeter  on [apt] repository 

### ISSUE THREAD

**Issue by Jukoo** [grassmunk/Chicago95#117]
the packet *lightdm-webkit-greeter** is not available in apt repositories ... but an equivalent is available **lightdm-gtk-greeter**

**Comment by EMH-Mark-I** [comment]
Yeah, this something I've noticed too. I haven't had the time to work on the greeter theme yet; I'm still working on the other components of the theme first in preparation for the next release.

**Comment by rchildre3** [comment]
For onlookers and possible path forward: current directions recommend `lightdm-webkit-greeter
` which depends on [deprecated webkit1](https://lists.webkit.org/pipermail/webkit-gtk/2014-March/001821.html) is [not being supported in Ubuntu going forward](https://packages.ubuntu.com/search?keywords=lightdm-webkit-greeter&searchon=names&suite=all&section=all). WebKitGTK2 exists and [Antergos has a greeter  that uses it](https://github.com/antergos/web-greeter) but now that [Antergos is discontinued](https://distrowatch.com/table.php?distribution=antergos), the development may stall, [and it has fairly consistent security issues](https://webkitgtk.org/news.html)

So it may be worth investigating [one of the recommended greeters on the lightdm README](https://github.com/canonical/lightdm/blob/master/README.md) possibly both GTK based and Qt based (For the experimental KDE support) greeters in the new instructions.

**Comment by grassmunk** [comment]
Honestly, the webkit greeter was always kinda a hack and not very good. The SDDM greeter is more accurate but I'm not a fan of telling people to install SDDM just to get a nice greeter. 

The future might have us writting our own based on https://www.freedesktop.org/wiki/Software/LightDM/Development/ and using GTK but we'll have to research what that would take. I'd be okay removing the current greeter and removing it from the instructions if its no longer applicable.

**Comment by EMH-Mark-I** [comment]
OK. That's fine with me too.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by BalooRJ1** [comment]
Just as a heads-up for anyone reading this, the LightDM theme created for the XFCE4-screensaver will also theme the Lightdm-GTK-Greeter. It looks pretty good. No need to install the webkit greeter.
